dashboard/test/models/ability_test.rb

```diff
@@ -3,7 +3,10 @@
 class AbilityTest < ActiveSupport::TestCase
   self.use_transactional_test_case = true
   setup_all do
-    @public_teacher_to_student_unit_group = create(:unit_group, instructor_audience: SharedCourseConstants::INSTRUCTOR_AUDIENCE.teacher, participant_audience: SharedCourseConstants::PARTICIPANT_AUDIENCE.student)
+    @public_teacher_to_student_unit_group = create(:unit_group, instructor_audience: SharedCourseConstants::INSTRUCTOR_AUDIENCE.teacher, participant_audience: SharedCourseConstants::PARTICIPANT_AUDIENCE.student) do |unit_group|
+      CourseOffering.add_course_offering(unit_group)
+      @reference_guide = create(:reference_guide, course_version: unit_group.course_version)
```

Comment:
It would be good to add a reference guide to one of the units with different audiences such as @public_facilitator_to_teacher_unit and test the ability to view that.